src/sentry/data_export/tasks.py

```diff
@@ -234,27 +231,13 @@ def process_rows(processor, data_export, batch_size, offset):
 
 @handle_snuba_errors(logger)
 def process_issues_by_tag(processor, limit, offset):
-    gtv_list_unicode = processor.get_serialized_data(limit=limit, offset=offset)
-    # TODO(python3): Remove next block once the 'csv' module has been updated
-    # to Python 3
-    if six.PY2:
-        gtv_list = convert_to_utf8(gtv_list_unicode)
```

Comment:
Looks like you can remove `convert_to_utf8` completely.

Reply:
Yeah this whole module can be refactored a bit to be simpler